### PR TITLE
Update registry.d with a REGSAM.KEY_ALL_ACCESS example...

### PR DESCRIPTION
**Pull request by notna123**
Adding example from http://forum.dlang.org/post/bkdvcwzaaknaxwvepbqf@forum.dlang.org

### CODE REVIEW

File: std/windows/registry.d [hunk 1]
```diff
@@ -1052,6 +1052,25 @@ public:
         Throws:
             If a value corresponding to the requested name is not found,
             $(D RegistryException) is thrown.
+
```
Comment (CyberShadow) [inline std/windows/registry.d]: Trailing whitespace, autotester complains

File: std/windows/registry.d [hunk 2]
```diff
@@ -1052,6 +1052,25 @@ public:
...
+            {
+                auto regKey = Registry.currentUser()
+                .getKey("Software")
+                .getKey("Microsoft")
+                .getKey("Windows")
+                .getKey("CurrentVersion")
+                .getKey("Run", REGSAM.KEY_ALL_ACCESS);
+                
+                regKey.setValue("Calculator", "calc.exe");
+                }
```
Comment (CyberShadow) [inline std/windows/registry.d]: This looks misindented

**Comment by CyberShadow** [comment]
I think this should go into the module documentation, or that of `REGSAM`, or that of `getKey`, but certainly not this particular overload of `setValue` which accepts string arguments.

File: std/windows/registry.d [hunk 3]
```diff
@@ -1052,6 +1052,24 @@ public:
         Throws:
             If a value corresponding to the requested name is not found,
             $(D RegistryException) is thrown.
+
```
Comment (CyberShadow) [inline std/windows/registry.d]:
```
Error - file 'std/windows/registry.d' contains trailing whitespace at line 1055
```

**Comment by CyberShadow** [comment]
Ping

**Comment by quickfur** [comment]
ping
The autotester is failing. Please fix?

**Comment by notna123** [comment]
first, sorry, I'm a bit lost with this...

1.     the autotester gives an "Error - file 'std/windows/registry.d' contains trailing whitespace at line 1067" i.e. here: https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=1970417&isPull=true
2.     but if I look into https://github.com/notna123/phobos/blob/b78b7f85416070142b251a3249d0db1d33a24a91/std/windows/registry.d there is no "trailing whitespace" at all in line 1067, only some "leading" ones --> "                .getKey("Run", REGSAM.KEY_ALL_ACCESS)"

**Comment by CyberShadow** [comment]
It's actually the next line, 1068. It contains nothing but whitespace (thus you could say that it has both leading and trailing whitespace). Probably the whitespace check program counts lines from 0.

**Comment by quickfur** [comment]
yes, looks like it's the blank line (1068) that needs to be trimmed so that there are no trailing spaces.

**Comment by DmitryOlshansky** [comment]
I concur - comment should be a general example at the top of the module

**Comment by notna123** [comment]
Hi there.
So whats next now? Do you need anything else from my side? If not, could you merge that one so I can remove it on my end?!
Thanks & regards

**Comment by CyberShadow** [comment]
See my & Dmitry's comment.

**Comment by notna123** [comment]
I don't get your point, sorry. I've just added an example because Walter tries to encourage ppl to do so and Dlang does really need many more of those...
If you think this example is added in the wring please, feel free to add it yourself and/or accept the merge and move it afterward to whatever place you think it belongs better. I don't care, as long as Dlangs gets more and better documentation and (best case copy-paste) examples...

Bye...

**Pull request closed without merging**